biz.aQute.bndlib/src/aQute/bnd/osgi/Verifier.java

```diff
@@ -84,7 +84,7 @@ public String toString() {
 
 	public final static Pattern	ReservedFileNames				= Pattern
 																		.compile(
-																				"CON(\\..+)?|PRN(\\..+)?|AUX(\\..+)?|CLOCK$|NUL(\\..+)?|COM[1-9](\\..+)?|LPT[1-9](\\..+)?|"
+																				"CON(\\..+)?|PRN(\\..+)?|AUX(\\..+)?|CLOCK\\\\$|NUL(\\..+)?|COM[1-9](\\..+)?|LPT[1-9](\\..+)?|"
 																						+ "\\$Mft|\\$MftMirr|\\$LogFile|\\$Volume|\\$AttrDef|\\$Bitmap|\\$Boot|\\$BadClus|\\$Secure|"
```

Comment:
Actually I think the logic is pretty broken. On line 85 of Verifier, the literal string is compiled to a Pattern... then we take the string representation of the pattern, macro-interpolate it, then recompile the result as a Pattern again. I think we should skip the first compile at least.